examples/verified-accounts/accounts.pact

```diff
@@ -56,12 +56,13 @@
       ))
 
   (defun read-account-admin (id)
-    (meta "Read data for account ID, admin version"
+    @doc "Read data for account ID, admin version"
+    @model
       (property
         (when
           (not (authorized-by 'accounts-admin-keyset))
           abort)
-        ))
```

Comment:
Can I ask about the value of this proof? We had discussed in the past that if the code enforces a keyset, the proof ends up duplicating that code which isn't worth much. As this module is presumably about best practices it would seem bad to do this.

Reply:
Aside: It's worth noting that this is changed a bit over in #182, where the property ends up looking like `(property (authorized-by 'accounts-admin-keyset))` (because we can't currently talk about `abort` in our property language, so the property we're looking at isn't current).

To address your point, I think this property is really useful once we can do this at a module level. It would be great to say this property or similar authorization logic holds over all the code in a module. A scenario I have in mind is that you'd like to say "this is how the public interface is authorized and this is how the admin interface is authorized". This is coming in #125 (which I'd love to get started on soon so feedback please!).

Reply:
Definitely useful at module level, or even as a `defproperty` reproduced here.